### ISSUE TITLE
Redesign of Details navigation

### ISSUE THREAD

**Issue by tschaka1904** [Complex-Portal/complex-portal-view#175]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search existing issues for a similar problems/requests
[X] feature request
[ ] support request => Please do not submit support request here, instead see: http://www.ebi.ac.uk/support/intact
```

**Current behaviour**

The current implementation of the 'GoTo'-Menu doesn't really work with the user, as #121 for example describes. Also, the general usage of the navigation is not that high. The questions which need to be asked is: 

- Do we need a navigation? (Most complexes are rather small)
- What do we want to show our user with it?  
- What kind of navigation do we want to implement? (Foundations magellan?)
- What is the benefit for the user? 

One idea I had, was removing the navigation completely. The 'GoTo'-Menu doesn't work everything else is stealing space or has sometimes some funny behaviour. Foundations magellan looks great, but it has its bugs and even on the Foundation website it is not working properly: 
![foundation_magellan_issue](https://user-images.githubusercontent.com/5269116/31219896-9ae28efa-a9b6-11e7-8184-7107efae4ba1.gif)

To be fair, this is Safari (Version 9.1 (11601.5.17.1)) and using Chrome and Firefox are behaving. 

Maybe we could also just make on the top visible what is there and what is not there. 

Basically, a list or row of all the sections marked as available or not available. By clicking you get to the section. Wit won't follow you around the page, but it indicates what the user could see in a different complex. 

Just making this ticket, as we probably won't be able to discuss this anymore in person and as a little bookmark.

**Comment by bmeldal** [comment]
Moved to Jira:
https://intact.atlassian.net/browse/CP-219
